tests/jballerina-unit-test/src/test/resources/test-src/balo/test_balo/annotations/annotation.bal

```diff
@@ -17,25 +14,3 @@ function testNonBallerinaAnnotations() returns foo:SomeConfiguration? {
     var tDesc = typeof someFunction;
     return tDesc.@foo:ConfigAnnotation;
 }
-
-@http:ServiceConfig {
```

Comment:
Why are we removing service tests? We should be able to use `lang.test:MockListener`, right?

There are some util methods to access resource annotations in `org.ballerinalang.nativeimpl.jvm.servicetests.ServiceValue` too. You can use interop to call these methods similar to test-src/services/service_decl.bal.

Reply:
If adding tests here could increase the test coverage, then definitely we should do that.

I think this deleted scenarios are invalid now.

Reply:
Given that this seems to be the only balo test we have for annotations IMO it would be better to test more than one scenario. Agreed that this particular test may not increase coverage but maybe we can have another test for services with an annotation with more than one attach point.

Reply:
Oh you mean to use a annotation from `balo`, +1 for that

Reply:
Create a separate issue for this https://github.com/ballerina-platform/ballerina-lang/issues/28699